src/components/ReleaseMonitor/__tests__/ReleaseMonitor.spec.tsx

```diff
@@ -0,0 +1,39 @@
+import '@testing-library/jest-dom';
+import { render, screen, waitFor } from '@testing-library/react';
+import * as k8s from '../../../k8s';
+import { mockNamespaceHooks } from '../../../unit-test-utils/mock-namespace';
+import { mockReleases, mockNamespaceData, mockNamespaces } from '../__data__/mock-release-data';
+import ReleaseMonitor from '../ReleaseMonitor';
+
+const mockUseNamespaceInfo = mockNamespaceHooks('useNamespaceInfo', mockNamespaceData);
+
+jest.mock('react-router-dom', () => ({
+  Link: (props) => <a href={props.to}>{props.children}</a>,
+}));
+
+describe('ReleaseMonitor', () => {
+  it('should render all columns', async () => {
+    mockUseNamespaceInfo.mockReturnValue({
```

Comment:
How about try some mock utils in our existing test suite? something like createK8sUtilMock or createK8sWatchResourceMock?  you need need 2-3 lines to get a mock.